frontend/packages/otr/public/mockServiceWorker.js

```diff
@@ -0,0 +1,340 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
```

Comment:
Miten tämä filu on luotu? Jos msw:n versiota päivitetään, tarvitaanko jotain erillisiä toimenpiteitä tämän tiedoston päivittämiseen?

Reply:
MSW luo mock workerin automaattisesti kun ajaa `npx msw init public/ --save`. Voisin laittaa tämän README.md

Reply:
itse asiassa jos laittaa vielä jonkun viittauksen package.jsoniin niin sitten kaikki päivittyy automaattisesti:

`That way any updates to the worker script will be automatically applied when updating the msw package in the future.`

Reply:
Tuon seurauksena pitäisi kyllä myös package.json:iin tulla seuraava entry:
`"msw": {"workerDirectory": "public"}`

Onko tämä vain jäänyt commitoimatta?

Reply:
> itse asiassa jos laittaa vielä jonkun viittauksen package.jsoniin niin sitten kaikki päivittyy automaattisesti:
> 
> `That way any updates to the worker script will be automatically applied when updating the msw package in the future.`

Jep, tätä ihmettelinkin.

Reply:
joo, nyt on kommitoitu